tests/model_preparation/test_reinsurance.py

```diff
@@ -3055,10 +3047,6 @@ def test_single_pol_level_fac_with_direct_layers(self):
             ri_info_df,
             ri_scope_df)
 
-        ri_inputs[0].ri_inputs.fm_policytc.to_csv('/tmp/fm_policytc.csv', index=False)
-        ri_inputs[0].ri_inputs.fm_programme.to_csv('/tmp/fm_programme.csv', index=False)
-        ri_inputs[0].ri_inputs.fm_profile.to_csv('/tmp/fm_profile.csv', index=False)
-
```

Comment:
These look like some debugging that was never deleted. They are not necessary for the tests to pass.

Worse than leaving temporary files lying around, they prevent two users from running this test on a shared machine. The test will fail for all but the first user.
1. User A runs the test which creates `/tmp/fm_....csv` owned by A.
2. User B run the test which tries to create the same files but can't because they already exist and are only (over)writeable by user A. Test fails.